android/src/main/java/com/swmansion/rnscreens/gamma/tabs/TabsHost.kt

```diff
@@ -94,7 +95,7 @@ class TabsHost(
 
     private val containerUpdateCoordinator = ContainerUpdateCoordinator()
 
-    private val wrappedContext = ContextThemeWrapper(reactContext, R.style.custom)
+    private val wrappedContext = ContextThemeWrapper(reactContext, com.google.android.material.R.style.Theme_Material3_DayNight_NoActionBar)
```

Comment:
I'm interested in what happens if we use these symbols inside our custom XML resources. Will they still trigger warnings? Maybe this is the solution. I'm talking about all symbols here: colours, font sizes, themes, etc.

Right now, I think we should proceed with the changes you suggested in this PR. We should experiment with custom xmls, however. Would you mind creating a ticket for this?